charts/seed-monitoring/charts/core/charts/prometheus/templates/config.yaml

```diff
@@ -448,6 +448,26 @@ data:
       metric_relabel_configs:
 {{ include "prometheus.keep-metrics.metric-relabel-config" .Values.allowedMetrics.nodeExporter | indent 6 }}
 
+    - job_name: kube-proxy
+      honor_labels: false
+      kubernetes_sd_configs:
+      - role: endpoints
+        api_server: https://kube-apiserver:443
+        tls_config:
+{{ include "prometheus.tls-config.kube-cert-auth" . | indent 10 }}
+      relabel_configs:
+      - source_labels:
```

Comment:
do we also need the service label here? 
```
      - action: labelmap
        regex: __meta_kubernetes_service_label_(.+)
```

Reply:
No, it will only add extra labels to our timeseries that don't add extra meaning.